server/config/urls.py

```diff
@@ -18,4 +19,5 @@
 
 urlpatterns = [
     url(r'^admin/', admin.site.urls),
+    url(r'^index/', TemplateView.as_view(template_name='index.html')),
```

Comment:
@ranmoshe The `^api` is including the `urls.py` from this path: `open_pension/server/api/urls.py`.